### PR TITLE
feat: x days left

### PR DESCRIPTION
**Pull request by fandihungryhub**
https://3.basecamp.com/5190892/buckets/31702802/card_tables/cards/5981068802

### CODE REVIEW

File: components/restaurant_detail/TextDaysLeft.vue [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="px-[5px] bg-[#0009] w-fit rounded">
```
Comment (afif-hh) [inline components/restaurant_detail/TextDaysLeft.vue:2]: warna nya jgn di hardcode, coba di cek dulu di tailwindconfig file

File: components/restaurant_detail/TextDaysLeft.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const props = defineProps({
+  date: {
+    type: String,
+    required: true,
+  },
+});
+
+const daysLeft = computed(() => {
+  const date = Date.parse(props.date);
```
Comment (afif-hh) [inline components/restaurant_detail/TextDaysLeft.vue:19]: pakai gini aja mas

`const dayjsInstance = dayjs(props.date, "format-string-nya-(misal yyyy-mm-dd)")
if(!dayjsInstance.isValid()){
return "Invalid date"
}
`

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: components/restaurant_detail/TextDaysLeft.vue (hunk 2 above)
Comment (fandihungryhub) [reply components/restaurant_detail/TextDaysLeft.vue]: updated

File: components/restaurant_detail/TextDaysLeft.vue (hunk 1 above)
Comment (fandihungryhub) [reply components/restaurant_detail/TextDaysLeft.vue:2]: di tailwindconfig gak ada, mau di tambahin?

Comment (afif-hh) [reply components/restaurant_detail/TextDaysLeft.vue:2]: iya mas boleh

File: components/restaurant_detail/TextDaysLeft.vue [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<script setup lang="ts">
+import { computed } from "vue";
+import dayjs from "~/lib/dayjs";
+
+const props = defineProps({
+  date: {
+    type: String,
+    required: true,
+  },
+});
```
Comment (afif-hh) [inline components/restaurant_detail/TextDaysLeft.vue:16]: kurang `defineOptions()`

Comment (fandihungryhub) [reply components/restaurant_detail/TextDaysLeft.vue:20]: updated

**Review by afif-hh — APPROVED** [review]

**Pull request merged**